docs/modules/operation/nav.adoc

```diff
@@ -127,8 +127,9 @@
 ** xref:application-perspective-monitoring/application-perspective-monitoring.adoc[]
 
 * xref:performance-data-collection/introduction.adoc[]
-** xref:performance-data-collection/collection-packages.adoc[]
+** xref:performance-data-collection/collectd/collection-packages.adoc[]
 ** xref:performance-data-collection/collectd/configuration.adoc[]
+** xref:performance-data-collection/resource-types.adoc[]
 ** xref:performance-data-collection/collectors/collectors.adoc[]
 *** xref:performance-data-collection/collectors/snmp/SnmpCollector.adoc[]
```

Comment:
I would suggest moving the SnmpCollector file into the same folder as the rest of the collectors and eliminating the snmp folder.

Reply:
You read my mind. I was going to do this, and realized it might break links. But probably better to fix it!

Reply:
@mmahacek , updated and alphabetized.